script/release/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.17.7 AS build
+
+ARG CONTAINERD_VERSION=1.5.7
+ARG NYDUS_BUILER_VERSION=v2.0.0-rc.2
+
+# Install acceld
+ADD ./ /accel
+RUN make -C /accel install-check-tools
+RUN make -C /accel
+
+# Install containerd
+RUN wget https://github.com/containerd/containerd/releases/download/v$CONTAINERD_VERSION/containerd-$CONTAINERD_VERSION-linux-amd64.tar.gz
+RUN mkdir -p /usr/local/bin/ && tar xzvf containerd-$CONTAINERD_VERSION-linux-amd64.tar.gz -C /usr/local/
+RUN ls -l /usr/local/bin/
+
+# Install nydus component
+RUN wget https://github.com/dragonflyoss/image-service/releases/download/$NYDUS_BUILER_VERSION/nydus-static-$NYDUS_BUILER_VERSION-x86_64.tgz
+RUN tar xzvf nydus-static-$NYDUS_BUILER_VERSION-x86_64.tgz && mv nydus-static/nydus-image /usr/local/bin/.
```

Comment:
Can we check the hash (sha256sum) of the tarballs?

Reply:
Thanks for the suggestion, will add the sha256sum file for nydus release first.

Reply:
@ktock Fixed, added sha256sum check.